### PR TITLE
faq: update descriptions about deploying TiDB on cloud

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version): done in #4005
- [ ] v4.0 (TiDB 4.0 versions): done in #4005
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/4005

### CODE REVIEW

File: faq/tidb-faq.md [hunk 1]
```diff
@@ -1018,15 +1018,15 @@ QPS 会统计执行的所有 SQL 命令，包括 use database、load data、begi
...
-## 八、Cloud TiDB
+## 八、云上部署
 
 ### 8.1 公有云
 
-#### 8.1.1 目前 Cloud TiDB 都支持哪些云厂商？
+#### 8.1.1 目前 TiDB 云上部署都支持哪些云厂商？
 
-Cloud TiDB 目前已经在腾讯云、UCloud 上线，都是数据库一级入口，欢迎大家使用。
+TiDB 云上部署目前已经在腾讯云、UCloud 上线，都是数据库一级入口，欢迎大家使用。
```
Comment (lilin90) [inline faq/tidb-faq.md:1027]: Please update it according to my suggestions on #4006.

Comment (TomShawn) [reply faq/tidb-faq.md]: Updated, PTAL.

**Comment by lilin90** [comment]
@TomShawn Please refer to [the English version](https://github.com/pingcap/docs/blob/master/faq/tidb-faq.md#deployment-on-the-cloud) and help update the descriptions in v3.1, v3.0, and v2.1. Thanks!

**Comment by TomShawn** [comment]
> @TomShawn Please refer to [the English version](https://github.com/pingcap/docs/blob/master/faq/tidb-faq.md#deployment-on-the-cloud) and help update the descriptions in v3.1, v3.0, and v2.1. Thanks!

Sure.

**Pull request merged**